examples/terachem_pbs.py

```diff
@@ -0,0 +1,25 @@
+import os
+import qcengine as qcng
+import qcelemental as qcel
+
+os.environ["TERACHEM_PBS_HOST"] = "127.0.0.1"
+os.environ["TERACHEM_PBS_PORT"] = "11111"
+
+prog = qcng.get_program("terachem_pbs")
+
+
+mol = qcel.models.Molecule.from_data(
+    """
+    O  0.0  0.000  -0.129
+    H  0.0 -1.494  1.027
+    H  0.0  1.494  1.027
+"""
+)
+
+inp = qcel.models.AtomicInput(
+    molecule=mol,
+    driver="energy",
+    model={"method": "pbe0", "basis": "6-31g"},
+)
+ret = prog.compute(inp)
+print(ret)
```

Comment:
I didn't remember that there was an examples/ directory. :-)

Go ahead and add terachem pbs to https://github.com/MolSSI/QCEngine/blob/master/qcengine/testing.py#L142 and add some test cases to qcengine/programs/tests/ with the `@using("terachempbs")` guard. At a minimum, it'd be  good to add terachempbs to https://github.com/MolSSI/QCEngine/blob/master/qcengine/tests/test_harness_canonical.py#L13 and https://github.com/MolSSI/QCEngine/blob/master/qcengine/tests/test_harness_canonical.py#L82 lists. This harness can't be covered in CI, but at least you and perhaps others will notice problems locally in pytest.

Reply:
Will do!

Reply:
@loriab not quite sure what to do here... Since you are unlikely to have a version of TeraChem running in the protocol buffer mode on your test server, should I add these tests? It seems you'd be unable to run them! Also, since the `tcpb` package is required, that's another stopper for running your tests against the TeraChem protocol buffer mode, unless we want to include the `tcpb` package as a requirement for `qcengine`, which we decided we didn't. I agree some tests of some sort would be good! But I'm not familiar enough with the package to see where to include tests that don't require actual computation or the additional `tcpb` package. Let me know your thoughts--thanks :)